examples/doublemach-mpi.py

```diff
@@ -0,0 +1,491 @@
+        },
+        mpi_communicator=comm
+    )
+    nodes = thaw(discr.nodes(), actx)
+
+    if use_overintegration:
+        quadrature_tag = DISCR_TAG_QUAD
+    else:
+        quadrature_tag = None  # noqa
+
+    dim = 2
+    if logmgr:
+        logmgr_add_device_name(logmgr, queue)
+        logmgr_add_device_memory_usage(logmgr, queue)
+        logmgr_add_many_discretization_quantities(logmgr, discr, dim,
+                             extract_vars_for_logging, units_for_logging)
+
+        logmgr.add_watches([
+            ("step.max", "step = {value}, "),
+            ("t_sim.max", "sim time: {value:1.6e} s\n"),
+            ("min_pressure", "------- P (min, max) (Pa) = ({value:1.9e}, "),
+            ("max_pressure",    "{value:1.9e})\n"),
+            ("min_temperature", "------- T (min, max) (K) = ({value:1.9e}, "),
+            ("max_temperature",    "{value:1.9e})\n"),
+            ("t_step.max", "------- step walltime: {value:6g} s, "),
+            ("t_log.max", "log walltime: {value:6g} s")
+        ])
+
+    # Solution setup and initialization
+    s0 = -6.0
+    kappa = 1.0
+    alpha = 2.0e-2
+    # {{{ Initialize simple transport model
+    kappa_t = 1e-5
+    sigma_v = 1e-5
+    # }}}
+
+    initializer = DoubleMachReflection()
+
+    from mirgecom.gas_model import GasModel, make_fluid_state
+    transport_model = SimpleTransport(viscosity=sigma_v,
+                                      thermal_conductivity=kappa_t)
+    eos = IdealSingleGas()
+    gas_model = GasModel(eos=eos, transport=transport_model)
+
+    def _boundary_state(discr, btag, gas_model, state_minus, **kwargs):
+        actx = state_minus.array_context
+        bnd_discr = discr.discr_from_dd(btag)
+        nodes = thaw(bnd_discr.nodes(), actx)
+        return make_fluid_state(initializer(x_vec=nodes, eos=gas_model.eos,
+                                            **kwargs), gas_model)
+
+    boundaries = {
+        DTAG_BOUNDARY("ic1"):
+        PrescribedFluidBoundary(boundary_state_func=_boundary_state),
+        DTAG_BOUNDARY("ic2"):
+        PrescribedFluidBoundary(boundary_state_func=_boundary_state),
+        DTAG_BOUNDARY("ic3"):
+        PrescribedFluidBoundary(boundary_state_func=_boundary_state),
+        DTAG_BOUNDARY("wall"): AdiabaticNoslipMovingBoundary(),
+        DTAG_BOUNDARY("out"): AdiabaticNoslipMovingBoundary(),
+    }
+
+    if rst_filename:
+        current_t = restart_data["t"]
+        current_step = restart_data["step"]
+        current_cv = restart_data["cv"]
+        if logmgr:
+            from mirgecom.logging_quantities import logmgr_set_time
+            logmgr_set_time(logmgr, current_step, current_t)
+    else:
+        # Set the current state from time 0
+        current_cv = initializer(nodes)
+    current_state = make_fluid_state(cv=current_cv, gas_model=gas_model)
+
+    visualizer = make_visualizer(discr,
+                                 discr.order if discr.dim == 2 else discr.order)
```

Comment:
`.order` is deprecated (it appears)

Reply:
I think that was used for a bit to plot at a higher discretion order but looks like I changed that and never got rid of the if statement. That part can be removed.

Reply:
Oh sure - but this particular line says something like "if sthing then discr.order else discr.order"

Reply:
[7b9ee93](https://github.com/illinois-ceesd/mirgecom/pull/582/commits/7b9ee9348c70a3424ac40369b942790bf4524aa8)